roles/installer/tasks/main.yml

```diff
@@ -4,6 +4,22 @@
     that:
       - foreman_installer_scenario is defined
 
+- when:
+    - ansible_distribution_major_version == '8'
+    - ansible_os_family == 'RedHat'
+  block:
+    - name: Get foreman-installer version
+      package:
+        list: foreman-installer
+      register: package_info
+
+    - name: Enable ruby:2.7 module
+      dnf:
+        name: '@ruby:2.7'
+        state: present
+      when:
+        - package_info.results[0]['version'] == 'nightly' or package_info.results[0]['version'] is version('2.5', '>=')
```

Comment:
The RPM version will never be "nightly" ;)

Reply:
and my example above used map/max to find the newest available version, just to be super sure for setups that have multiple repos enabled…